contracts/allocators/StakingManager.sol

```diff
@@ -0,0 +1,94 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+pragma solidity ^0.8.10;
+
+import "../libraries/SafeERC20.sol";
+
+import "../interfaces/IERC20.sol";
+import "../interfaces/ITreasury.sol";
+
+import "../types/OlympusAccessControlled.sol";
+
+interface IStaking {
+    function stake( uint _amount, address _recipient ) external returns ( bool );
+    function unstake( uint _amount, bool _trigger ) external;
+    function claim ( address _recipient ) external;
+}
+
+contract OlympusStakingManager is OlympusAccessControlled {
```

Comment:
Should prob rename to something more specific. The name sounds general but this contract is specifically for Lobis and Redacted. Maybe `MetaGovernanceAllocator` ?

Reply:
done